MAINTAINERS.md

```diff
@@ -27,16 +29,25 @@ Review pull requests regularly, comment, suggest, reject, merge and close. Accep
 
 ### Triage Open Issues
 
-Review issues regularly and triage by labelling issues. For example, add "help wanted" to good issues for new community members. Request for more information from a submitter if an issue is not clear. Create new labels as needed by the project.
+Manage labels, review issues regularly, and triage by labelling them. For example, add "help wanted" to good issues for new community members and *blocker* for issues that scare you or need immediate attention. Request for more information from a submitter if an issue is not clear. Create new labels as needed by the project.
+
+### Be Responsive
+
+Respond to enhancement requests, and forum posts. Allocate time to reviewing and commenting on issues and conversations as they come in. 
 
 ### Maintain Overall Health of the Repo
 
-Keep the main branch at production quality at all times.
+Keep the `main` branch at production quality at all times. Backport features as needed. Cut release branches and tags to enable future patches. 
+
+### Use Semver
+
+Use and enforce [semantic versioning](https://semver.org/) and do not let breaking changes be made outside of major releases.
 
 ### Release Frequently
 
 Make frequent project releases to the community.
```

Comment:
Again -- how?  I think we need to write down how you release and when we plan to do it.

Reply:
This should say "see releasing", but since it doesn't exist we can't spell it out now.